src/uniprotkb/components/graphs/PieChart.tsx

```diff
@@ -29,6 +29,19 @@ const margin = 45;
 
 const radius = Math.min(width, height) / 2 - margin;
 
+// Function to distributre pie chart data to hopefully avoid label collisions
+export const distributeByEntryCount = (data: StatisticsGraphItem[]) => {
+  const sorted = data.sort((a, b) => a.entryCount - b.entryCount);
```

Comment:
```suggestion
  const sorted = new Array(data).sort((a, b) => a.entryCount - b.entryCount);
```
Data is coming from the props, in order to avoid unexpected issues it's better to make a copy of the array as sort will mutate the original one

Reply:
That behaviour surprises me! Good to know!

Reply:
It makes sense actually as it's an instance method. I noticed the tests failed so I used a spread instead.

Reply:
Sorry I've just used the GitHub suggestion and didn't realise it wasn't the proper thing to use, I meant to use `Array.from()` which is essentially the same as what you used with the array spreading